### DISCUSSION TITLE
Group follow(ers/ing)

### DISCUSSION THREAD

**Discussion by doguedogue** [community/community#18901] [Other Feature Feedback, Questions, & Ideas]
GitHub, can you add a mechanism to group people, followers /following?

Sometimes you forget from where you add people (maybe only me) 

Or something to tag a description for people.

**Comment by ghost** [comment]
@doguedogue Hi! How are you? Your idea is amazing, I was even thinking of something similar here: https://github.com/github-community/community/discussions/16748 - The difference is that the group of followers would be in the discussions, where I can say that a group of members is admin, manager, etc. In other words, the group of followers has specific functions.
